pages/submissions/new.tsx

```diff
@@ -0,0 +1,56 @@
+import { GetServerSideProps } from 'next';
+import StartForm, { FormValues } from 'components/StartForm/StartForm';
+import { useRouter } from 'next/router';
+import { Form } from 'data/flexibleForms/forms.types';
+import { getProtocol } from 'utils/urls';
+import { FormikHelpers } from 'formik';
+import axios from 'axios';
+
+interface Props {
+  forms: Form[];
+}
+
+const NewSubmissionPage = ({ forms }: Props): React.ReactElement => {
+  const router = useRouter();
+
+  const handleSubmit = async (
```

Comment:
I think we want a `useCallback` here

Reply:
i understand callbacks and memos as a performance optimisation to prevent them from expensively re-running or causing needless re-renders.

i don't think either of those apply here - the function only runs when you hit the button (once). [this is close to my understanding](https://dmitripavlutin.com/dont-overuse-react-usecallback/)

Reply:
Yeah that's a fair point, I thought I had a specific reason for this one but now can't think what that was. Perhaps it was depending on whether `props.forms` changes frequently (as it's passed down to a child, I don't know how likely to change that is. If you think it's fairly static then I think fair to keep it as-is for now.